tests/torch/data/reference_graphs/const_sparsity/shuflenet_g2.dot

```diff
@@ -10,264 +10,280 @@ strict digraph  {
-"242 ShuffleNet/Sequential[layer3]/Bottleneck[2]/relu_1" [id=242, type=relu];
-"243 ShuffleNet/Sequential[layer3]/Bottleneck[2]/NNCFConv2d[conv3]/ModuleDict[pre_ops]/UpdateWeight[0]/BinaryMask[op]/apply_binary_mask_0" [id=243, type=apply_binary_mask];
-"244 ShuffleNet/Sequential[layer3]/Bottleneck[2]/NNCFConv2d[conv3]/conv2d_0" [id=244, type=conv2d];
-"245 ShuffleNet/Sequential[layer3]/Bottleneck[2]/NNCFBatchNorm[bn3]/batch_norm_0" [id=245, type=batch_norm];
-"246 ShuffleNet/Sequential[layer3]/Bottleneck[2]/__add___0" [id=246, type=__add__];
-"247 ShuffleNet/Sequential[layer3]/Bottleneck[2]/relu_2" [id=247, type=relu];
-"248 ShuffleNet/Sequential[layer3]/Bottleneck[3]/NNCFConv2d[conv1]/ModuleDict[pre_ops]/UpdateWeight[0]/BinaryMask[op]/apply_binary_mask_0" [id=248, type=apply_binary_mask];
-"249 ShuffleNet/Sequential[layer3]/Bottleneck[3]/NNCFConv2d[conv1]/conv2d_0" [id=249, type=conv2d];
-"250 ShuffleNet/Sequential[layer3]/Bottleneck[3]/NNCFBatchNorm[bn1]/batch_norm_0" [id=250, type=batch_norm];
-"251 ShuffleNet/Sequential[layer3]/Bottleneck[3]/relu_0" [id=251, type=relu];
-"252 ShuffleNet/Sequential[layer3]/Bottleneck[3]/ShuffleBlock[shuffle1]/view_0" [id=252, type=view];
-"253 ShuffleNet/Sequential[layer3]/Bottleneck[3]/ShuffleBlock[shuffle1]/permute_0" [id=253, type=permute];
-"254 ShuffleNet/Sequential[layer3]/Bottleneck[3]/ShuffleBlock[shuffle1]/view_1" [id=254, type=view];
-"255 ShuffleNet/Sequential[layer3]/Bottleneck[3]/NNCFConv2d[conv2]/ModuleDict[pre_ops]/UpdateWeight[0]/BinaryMask[op]/apply_binary_mask_0" [id=255, type=apply_binary_mask];
-"256 ShuffleNet/Sequential[layer3]/Bottleneck[3]/NNCFConv2d[conv2]/conv2d_0" [id=256, type=conv2d];
-"257 ShuffleNet/Sequential[layer3]/Bottleneck[3]/NNCFBatchNorm[bn2]/batch_norm_0" [id=257, type=batch_norm];
-"258 ShuffleNet/Sequential[layer3]/Bottleneck[3]/relu_1" [id=258, type=relu];
-"259 ShuffleNet/Sequential[layer3]/Bottleneck[3]/NNCFConv2d[conv3]/ModuleDict[pre_ops]/UpdateWeight[0]/BinaryMask[op]/apply_binary_mask_0" [id=259, type=apply_binary_mask];
-"260 ShuffleNet/Sequential[layer3]/Bottleneck[3]/NNCFConv2d[conv3]/conv2d_0" [id=260, type=conv2d];
-"261 ShuffleNet/Sequential[layer3]/Bottleneck[3]/NNCFBatchNorm[bn3]/batch_norm_0" [id=261, type=batch_norm];
-"262 ShuffleNet/Sequential[layer3]/Bottleneck[3]/__add___0" [id=262, type=__add__];
-"263 ShuffleNet/Sequential[layer3]/Bottleneck[3]/relu_2" [id=263, type=relu];
-"264 ShuffleNet/avg_pool2d_0" [id=264, type=avg_pool2d];
-"265 ShuffleNet/view_0" [id=265, type=view];
-"266 ShuffleNet/NNCFLinear[linear]/ModuleDict[pre_ops]/UpdateWeight[0]/BinaryMask[op]/apply_binary_mask_0" [id=266, type=apply_binary_mask];
-"267 ShuffleNet/NNCFLinear[linear]/linear_0" [id=267, type=linear];
-"268 /nncf_model_output_0" [id=268, type=nncf_model_output];
+"11 ShuffleNet/Sequential[layer1]/Bottleneck[0]/ShuffleBlock[shuffle1]/contiguous_0" [id=11, type=contiguous];
```

Comment:
Should remove the `ContiguousMetatype` and instead mark `contiguous` as a `NoopMetatype`.